build_early.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

I get a segementation fault, b/c 
```shellp
$ which bash
/usr/bin/bash

```